src/webparts/laserficheAdminConfiguration/components/ProfileConfigurationComponents.tsx

```diff
@@ -119,10 +119,10 @@ export function ConfigurationBody(props: {
   }
   return (
     <>
-      <div className='form-group row'>
+      <div className={`${styles.formGroupRow} form-group row`}>

```

Comment:
why keep both 'form-group row' and add styles.formGroupRow - do they not implement the same set of styles?

Reply:
form-group row are bootstrap styles, formGroupRow was another style I added on top of those to center the row